pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+requires = ["setuptools>=72", "wheel", "setuptools_scm[toml]>=8"]
+
+[project]
+name = "etos_suite_starter"
+dynamic = ["version"]
+description = "ETOS Suite Starter"
+authors = [{name = "Tobias Persson", email = "tobias.persson@axis.com"}]
+license = { text  = "Apache License, Version 2.0" }
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License"
+]
+
+dependencies = [
+    "etos_lib==4.3.1",
+    "opentelemetry-api~=1.21",
+    "opentelemetry-exporter-otlp~=1.21",
+    "opentelemetry-sdk~=1.21"
+]
+
+[options]
+zip_safe = false
+include_package_data = true
+python_requires = ">=3.4"
```

Comment:
Can we run with python 3.4?

Reply:
Most likely not. I've avoided changing the original value in this PR.

Reply:
Changed to >=3.9.